File: OddyseyServer/src/com/src/audio/AudioFile.java
package com.src.audio;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

//Estas son las canciones
@XmlRootElement(name = "Audio")
public class AudioFile {

	@XmlElement(name = "Name")
	public String name;
	@XmlElement(name = "Author")
	public String author;
	@XmlElement(name = "Album")
	public String album;
	@XmlElement(name = "Year")
	public String year;
	@XmlElement(name = "Lyrics")
	public String lyrics;
	@XmlElement(name = "Time")
	public String time;
	@XmlElement(name = "Score")
	public String score;
	@XmlElement(name = "Data")
	public String data;

	@SuppressWarnings("unused")
	private AudioFile() {
		
	}
	
	// Constructor
	public AudioFile(String name, String author, String album, String year, String lyrics, String time) {
		this.name = name;
		this.author = author;
		this.album = album;
		this.year = year;
		this.lyrics = lyrics;
		this.time = time;
	}
	
	//For debugging
	@Override
	public String toString() {
		return name + "/" + author + "/" + album + "/" + year + "/" + lyrics + "/" + time + "/" + score;
	}
	
	
}

File: Oddysey/Oddysey/Oddysey/AudioFile.cs
﻿using System;

namespace Oddysey
{
    public class AudioFile
    {
        public String Name { get; set; }
        public String Author { get; set; }
        public String Album { get; set; }
        public String Year { get; set; }
        public String Lyrics { get; set; }
        public String Time { get; set; }
        public String Score { get; set; }
        public byte[] Data { get; set; }

        public void SetMainParameters(String name, String author, String time)
        {
            Name = name;
            Author = author;
            Time = time;
        }

    }
}
